ODIN_II/SRC/parse_making_ast.cpp

```diff
@@ -1874,25 +1882,68 @@ void graphVizOutputAst_traverse_node(FILE* fp, ast_node_t* node, ast_node_t* fro
             break;
 
         case IDENTIFIERS:
-            fprintf(fp, ": %s", node->types.identifier);
+            fprintf(fp, " %s", node->types.identifier);
             break;
 
         default:
             break;
     }
-    fprintf(fp, ">];\n");
+
+    if (node->identifier_node || node->type == IDENTIFIERS || node->type == NUMBERS) {
+        fprintf(fp, ">];\n");
```

Comment:
i'd prefer if you wrapped everything in HTML brackets. makes it simpler and work just a well.
also allows us to put new lines

Reply:
Like the titles?